src/main/java/com/alextherapeutics/diga/implementation/DigaCodeDefaultParser.java

```diff
@@ -93,9 +94,23 @@ private ParsedDigaCode parseCodeString(String codeString) {
   }
 
   private boolean validateDigaCodeStructure(String code) {
-    // TODO - make more check, like regex, the different parts, etc.
```

Comment:
If I remember the documentation correctly the codes can consist of `[A-Z]` or `[2-7]`. How does the base32check and diga api behave if codes contain lower-cased characters, e.g. by typos when entering prescription codes? Would it be a good idea to always capitalize the codes within the diga-api-client?

Reply:
I would leave it to the client implementation to send the proper (uppercase) code

Reply:
Fair enough 👍